modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -762,6 +763,154 @@ public void testAddObjectEntry() throws Exception {
 				_simpleDTOConverterContext, _objectDefinition1,
 				parentObjectEntry1.getId()));
 
+		// Aggregation field without filters based on Many to Many relationship
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				_adminUser.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition1.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"manyToManyRelationshipName",
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		_addAggregationObjectField(
+			"countAggregationObjectFieldNameSelf", "COUNT", null,
```

Comment:
the test group is `Aggregation field without filters based on Many to Many relationship`, so the suffix `self` doesn't make sense here I believe the name should be:

`countManyToManyAggregationObjectFieldName`, `countM2MAggregationObjectFieldName` or `countAggregationObjectFieldName2`.

I believe the third one will be approved by Brian, the other ones I'm not sure, anyway the one you choose you will need to rename the old one as well, for example, if you choose `countAggregationObjectFieldName2` you will need to rename the old one to `countAggregationObjectFieldName1`.

Thank you!